Don't add empty user to roles

OliviaYtterbrink opened the pull request:
- If a role was added to the roles file manually the role was populated
  with a user with no name

lutovich commented on enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/RoleSerialization.java:
```diff
@@ -101,7 +101,10 @@ private String serialize( SortedSet<String> users )
         for ( String user : splits )
         {
-            users.add( user );
+            if (!user.trim().isEmpty())
```
Maybe use `org.apache.commons.lang3.StringUtils#isNotBlank(String)`

OliviaYtterbrink committed:
Don't add empty user to roles

- If a role was added to the roles file manually the role was populated
 with a user with no name
